### DISCUSSION TITLE
Termux error (now solved)

### DISCUSSION THREAD

**Discussion by u-8d** [aahnik/tgcf#80] [General]
![Screenshot_20210424_113006](https://user-images.githubusercontent.com/81148580/115949198-fb4a7800-a4f0-11eb-9ddb-3745a67409cd.jpg)

What error is this now😫

**Comment by aahnik** [comment]
give the full logs.

something happened during the `pkg upgrade`

Try running `pkg upgrade` manually., and then run the installation command.

You can run all the commands manually also.

i have added more details here -> https://github.com/aahnik/tgcf/wiki/Run-on-Android-using-Termux#install-tgcf-on-termux

**Comment by aahnik** [comment]
hi @chiefgamer what did you enter when the prompt appeared? did you press anything on keyboard?

**Comment by u-8d** [comment]
No nothing pressed I just run the cmd and automatically this all happened

**Reply by aahnik** [reply]
I updated the installer script. Please uninstall termux, and delete its cache files.

Then install termux, and try running `curl -Lks bit.ly/tgcf-termux | bash`

**Reply by u-8d** [reply]
Done 🤗 bro successfully installed 🙂👍

**Comment by aahnik** [comment]
hi, @chiefgamer , please test and comment.

**Reply by u-8d** [reply]
![Screenshot_20210424_164813](https://user-images.githubusercontent.com/81148580/115956869-ec79ba80-a51c-11eb-9c42-fced145f4f19.jpg)
